extension/content/firebug/lib/url.js

```diff
@@ -417,7 +417,7 @@ Url.normalizeURL = function(url)
 
     // Normalize path traversals (a/b/../c -> a/c).
     while (url.indexOf("/../") !== -1 && url[0] != "/")
-        url = url.replace(/[^\/]+\/\.\.\//g, "");
+        url = url.replace(/[^\/]+\/+\.\.\//g, "");
 
```

Comment:
Thanks for quick patch!

I am getting: http://prevenity.com/test
From: http://prevenity.com/?falsevariable=test;//;../../../test
While better would probably be: http://prevenity.com/

But, the original infinite loop is gone and that's what the report is about.
Honza